### PR TITLE
Genai page addition

### PR DESCRIPTION
**Pull request by MaanavD**
Revamped Gen AI page based on Discovery interns. 

Preview available at: https://maanavd.github.io/onnxruntime/generative-ai

### CODE REVIEW

**Comment by MaanavD** [comment]
screencapture-localhost-5173-generative-ai-2024-09-04-13_57_19

^Screenshot of entire page while we wait for fixing website publishing pipelines

**Comment by MaanavD** [comment]
@natke between "Gen-ai" or "Generative AI" and "ONNX Runtime" and "ORT", do we have a decisive answer on how we're marketing the page?

File: src/routes/generative-ai/+page.svelte [hunk 1]
```diff
@@ -1,86 +1,258 @@
...
-	let authors = ['']
-	let keywords = 'onnxruntime, onnx runtime generative ai, onnx runtime generative ai models, onnx runtime generative ai deployment, onnx runtime generative ai performance, onnx runtime generative ai time to market, onnx runtime generative ai deploy anywhere, onnx runtime generative ai boost performance, onnx runtime generative ai improve time to market, onnx runtime generative ai production ready, onnx runtime generative ai lower latency, onnx runtime generative ai higher throughput, onnx runtime generative ai get innovations into production faster, onnx runtime generative ai testimonials, onnx runtime generative ai performance enhancements, onnx runtime generative ai production ready, onnx runtime generative ai lower latency, onnx runtime generative ai higher throughput, onnx runtime generative ai get innovations into production faster, onnx runtime generative ai testimonials, onnx runtime generative ai performance enhancements'
+	let image = 'https://i.ibb.co/0YBy62j/ORT-icon-for-light-bg.png';
+	let imageSquare = 'https://i.ibb.co/0YBy62j/ORT-icon-for-light-bg.png';
+	let authors = [''];
+	let keywords =
```
Comment (natke) [inline src/routes/generative-ai/+page.svelte:19]: I don't think we need meta keywords - not used by major search engines anymore.

File: src/routes/generative-ai/+page.svelte [hunk 2]
```diff
@@ -1,86 +1,258 @@
...
-						href="https://onnxruntime.ai/docs/tutorials/csharp/stable-diffusion-csharp.html"
-						class="btn-primary btn">Inference Stable Diffusion →</a
-					>
+	<div class="my-8 grid grid-cols-1 md:grid-cols-4 gap-4">
+		<img class="m-auto rounded-xl" src={aibrain} alt="Representing generative AI">
+		<div class="col-span-3 my-auto">
+
+			<h2 class="text-3xl">What is Generative AI?</h2>
+			<p>
+				Generative AI refers to a type of artificial intelligence that creates new content—such as
```
Comment (natke) [inline src/routes/generative-ai/+page.svelte:58]:
```suggestion
Generative AI refers to a type of artificial intelligence that creates new content—such as text, images, audio, or code— in response to a user prompt or instruction. 
```

File: src/routes/generative-ai/+page.svelte [hunk 3]
```diff
@@ -1,86 +1,258 @@
...
+				text, images, audio, or code—based on patterns learned from existing data. Unlike
+				traditional AI models, which primarily classify or predict based on given inputs, generative
+				AI models produce entirely new outputs.
+				<br /><br />
+				They accomplish this through advanced techniques like deep learning, often using models such
+				as Transformers and Generative Adversarial Networks (GANs). Examples include AI that generates
+				human-like text responses, creates realistic images from descriptions, or composes music. Generative
+				AI is driving innovation across industries by enabling personalized experiences, automating creative
+				processes, and opening new possibilities for content generation!
```
Comment (natke) [inline src/routes/generative-ai/+page.svelte:67]:
```suggestion
Generative AI models accomplish this through deep learning, using for example, Transformers for text and Diffusion models for images. Examples include AI that generates human-like text responses, creates realistic images from descriptions, transcribes and summarizes audio, and composes music. Generative AI is driving innovation across industries by driving business productivity, enabling personalized experiences, automating creative processes, and opening new possibilities for content generation!
```

File: src/routes/generative-ai/+page.svelte [hunk 4]
```diff
@@ -1,86 +1,258 @@
...
+					<p>
+						Text generation models are AI systems designed to generate human-like text based on
+						prompts. They're used in chatbots, content creation, summarization, and creative
+						writing. Check out our phi-3 and mistral demos below:
+					</p>
+				</div>
+			
+				<div class="grid gap-4 grid-cols-1 md:grid-cols-2 mt-auto">
+					<a href="https://github.com/microsoft/onnxruntime-genai/blob/main/examples/python/phi-3-tutorial.md" class="btn btn-primary btn-block">Phi-3</a>
+					<a href="https://github.com/microsoft/onnxruntime-inference-examples/tree/main/python/models/mistral" class="btn btn-primary btn-block">Mistral</a>
```
Comment (natke) [inline src/routes/generative-ai/+page.svelte:91]: Are these meant to be links to models or links to tutorials?

File: src/routes/generative-ai/+page.svelte [hunk 5]
```diff
@@ -1,86 +1,258 @@
...
+			<!-- <div class="my-auto">
+				<a href="" class="btn btn-primary btn-block m-4">Run on Desktop</a>
+				<a href="" class="btn btn-primary btn-block m-4">Run on Mobile</a>
+				<a href="" class="btn btn-primary btn-block m-4">Run in Browser</a>
+			</div> -->
+			<div class="grid gap-4 grid-cols-1 md:grid-cols-2 lg:grid-cols-2 my-8">
+				<div class="card bg-base-300 text-neutral-content">
+					<div class="card-body items-center text-center">
+						<h2 class="card-title">Multiplatform</h2>
+						<p>Whether it be Desktop, Mobile, Browser, or the Cloud, run ONNX Runtime on the platform of your choosing!</p>
```
Comment (natke) [inline src/routes/generative-ai/+page.svelte:173]: Too many bangs (!)

File: src/routes/generative-ai/+page.svelte [hunk 6]
```diff
@@ -1,86 +1,258 @@
...
+		
 	</div>
-	<h2 class="text-3xl">Performance</h2>
-	<p class="pb-4">The average latency in seconds on Stable Diffusion v1.5 and v2.1 models:</p>
-	<div class="grid gap-10 grid-cols-1 md:grid-cols-2 pb-10">
-		<div class="">
-			<img src={stablediffusion1} alt="Stable Diffusion v1.5 latency graphs" />
-		</div>
+	<div class="">
+		<h3 class="text-2xl">Tutorials & Demos</h3>
```
Comment (natke) [inline src/routes/generative-ai/+page.svelte:199]: Include a web demo in this section?

File: src/routes/generative-ai/+page.svelte [hunk 7]
```diff
@@ -1,86 +1,258 @@
...
+	import genny from './genny.png';
+	import vision_ui from './vision_UI.png';
+	import mobile from './mobile.png';
+	import desktop from './desktop.png';
+	import cloud from './browser.png';
+	import aibrain from './aibrain.webp'
+	const title = 'Generative AI';
 	const description =
-		'Integrate the power of generative AI in your apps and services with ONNX Runtime. Broad platform support and deep optimizations empower usage of state-of-the-art models for image synthesis, text generation, and more.';
+		'Integrate the power of generative AI in your apps and services with ONNX Runtime. Broad platform support and deep optimizations empower usage of state-of-the-art models for text generation, audio synthesis, and more.';
```
Comment (natke) [inline src/routes/generative-ai/+page.svelte:13]: Do we want to keep the meta description and first para technology generic?

**Review by natke — COMMENTED** [review]
Look and feel is great! Adding some content suggestions

File: src/routes/generative-ai/+page.svelte (hunk 1 above)
Comment (MaanavD) [reply src/routes/generative-ai/+page.svelte:19]: It's passed in by every page right now - you're def right that it's not super used, but I don't think it hurts 😅 If you'd like we could discuss refactoring the website outright though

File: src/routes/generative-ai/+page.svelte (hunk 4 above)
Comment (MaanavD) [reply src/routes/generative-ai/+page.svelte:87]: links to usable models or tutorials - both should be fine as one is an example one is a tutorial

File: src/routes/generative-ai/+page.svelte (hunk 7 above)
Comment (MaanavD) [reply src/routes/generative-ai/+page.svelte:13]: will specify more directly in the meta

**Comment by MaanavD** [comment]
Images may not work on live preview (only an issue with preview). Here are images: 

image
image

**Review by natke — APPROVED** [review]

**Pull request merged**